Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2_test.py

```diff
@@ -89,3 +105,12 @@ def test_file_command(mocker, raw_response_general, raw_response_analysis, expec
     results = file_command(client, {'file': '6c5360d41bd2b14b1565f5b18e5c203cf512e493'})
     # results is tuple (human_readable, context_entry, raw_response).
     assert expected == results[1]
+
+
+@pytest.mark.parametrize('function, command_function, args, mocker_result, expected_result', URL_COMMAND)
```

Comment:
Why use parametrize if you only have one item in the list?
And please add here the Given When Then docstring:
![image](https://user-images.githubusercontent.com/50294648/113881514-2af54280-97c5-11eb-9b2a-c39ec9189518.png)
You can find it in many of our content Unit tests